src/components/settings/settings.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+
+import CardBackButton from '../card-back-button';
+import BoardSettingButton from '../board-setting-button';
+import CardTypeButton from '../card-type-button';
+
+import './settings.css';
+
+import blueBack from '../card/images/card-back-blue.png';
+import redBack from '../card/images/card-back-red.png';
+
+import cards from './images/cards.jpg';
+import pictures from './images/pictures.jpg';
+
+export default class Settings extends Component {
+  state = {
+    cardBacks: [
+      { color: 'blue', imgSrc: blueBack },
+      { color: 'red', imgSrc: redBack }
+    ],
+    boardSizes: [12, 18, 24],
+    cardTypes: [
+      { type: 'cards', imgSrc: cards },
+      { type: 'pictures', imgSrc: pictures }
+    ]
+  }
+  render() {
+    const { isSettingsOpen, applySettings, chooseSetting, choosenSettings } = this.props;
+    const { cardBacks, boardSizes, cardTypes } = this.state;
+
+    const clazz = isSettingsOpen ? 'settings--show' : '';
+
+    const cardBackButtons = cardBacks.map(({color, imgSrc}, idx) => {
+      return (<CardBackButton name={color}
+                              key={idx+1}
```

Comment:
можно ничего не делать с индексом, эффект будет тот же самый
```suggestion
                              key={idx}
```